first_ore_preference: [1, 2, 3, 4, 5]
locations:
  exchange_detection: [0.0948824, 0.860618, 0.444098]
  ore_1: [-0.0147564, 2.48456, 1.53682]
  ore_2: [-0.0835764, 2.31284, -0.0191219]
  ore_3: [1.83616, 2.72457, -0.0186984]
  ore_4: [2.38291, 0.171916, 3.1297]
  ore_5: [2.51022, -0.744519, -0.0609967]
  exchange_B: [1.05052, 1.9234, -0.134978]
  exchange_O: [1.05859, 1.80549, -0.0527012]
  exchange_X: [1.04136, 1.53727, 0.195646]
